examples/gitlab-ha/main.tf

```diff
@@ -21,13 +21,14 @@
  */
 
 provider "aws" {
-  region = "${var.region}"
+  region = var.region
 }
 
-data "aws_availability_zones" "available" {}
+data "aws_availability_zones" "available" {
+}
 
 data "aws_route53_zone" "zone" {
-  name = "${var.dns_zone_name}."
+  name         = "${var.dns_zone_name}."
```

Comment:
The trailing dot seems to be accidental here.

Reply:
I believe it was added assuming that the operator would provide `foo.bar.tld`, but we'd need to query with `foo.bar.tld.` - this is a DNS thing